gems/smithy/lib/smithy/model/rbs.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Model
+    # @api private
+    class Rbs
```

Comment:
Rbs or RBS as convention? I'm ok with either. I lean towards RBS. I think views it has to be Yml or Rbs.

Reply:
I've used `Rbs` everywhere so far because to me `ClientRbs` reads slightly better than `ClientRBS`.  But not a strong opinion and I'm happy to change all of them.

Reply:
I think RBS for this module and Rbs for views is fine

Reply:
Updated to RBS